index.bs

```diff
@@ -1285,6 +1295,17 @@ controls the maximum [=map/size=] of a [=trigger spec map=] for an
 the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=].
 Its value is 64.
 
+<dfn>Default max event states</dfn> is a positive integer that controls the
+default [=attribution source/max event states=]. Its value is 3.
+
+<dfn>Max length per attribution scope</dfn> is a positive integer that controls the
+maximum [=string/length=] of an attribution scope from [=attribution source=]'s [=attribution source/attribution scopes=] or [=attribution trigger=]'s [=attribution trigger/attribution scopes=].
+Its value is 50.
+
+<dfn>Max number of attribution scopes</dfn> is a positive integer that controls the
```

Comment:
IIUC, a trigger should be able to specify arbitrarily many scopes that it is interested in being matched against.

Reply:
Agreed with @apasel422, I don't see a privacy concern on specifying unlimited number of attribution scopes on the trigger side as long as the attribution scopes on the sources are within the limit.
The limit is needed on sources as the attribution scopes need to be persisted in storage.

Reply:
Good point. created a separate function for triggers as the logic is quite different now.